keyboards/handwired/tennie/tennie.c

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2018 Jack Hildebrandt
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "tennie.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+    rgblight_enable();
```

Comment:
```suggestion
```

Instead, it may be better to run: 
```c
void keyboard_post_init_kb(void) {
  rgblight_enable_noeeprom();
  keyboard_post_init_user();
}
```

This runs after the rgb light code has initialized, and will be properly retained.

Reply:
Alright, I'll switch to that then!